cmd/assignIlkLumpToZero.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/cat/v1_0_0"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	storage2 "github.com/makerdao/vulcanizedb/libraries/shared/storage"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
+	"github.com/makerdao/vulcanizedb/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	blockNumber int
+	ilks        []string
+)
+
+var assignIlkLumpToZeroCmd = &cobra.Command{
+	Use:   "assignIlkLumpToZero",
+	Short: "Create a diff with a zero ilk lump value.",
+	Long: `Inserts artificial cat_ilk_lump storage_diffs with storage_values of zero for the given ilks at the given block
+height. The storage_diffs will then be picked up in the execute process to be transformed into maker.cat_ilk_lump records.
+The block-number (b) and ilks (i) arguments are required.
+
+Note: the ilk argument(s) passed in need to be the hex representation of the ilk.
```

Comment:
I wanted to call this out - we'll need to pass the ilks in as their hex representation, i.e. `0x4241542d41000000000000000000000000000000000000000000000000000000`, `0x4554482d41000000000000000000000000000000000000000000000000000000`. 

Though not ideal, I thought it was easier than converting the ilkIdentifiers (i.e. `ETH-A`, `BAT-A`) into these hex representations. Another idea I had was to look up the ilks from the db, based on their identifiers. Not sure which is easiest/makes the most sense but up for some ideas!

Reply:
Totally on board with your reasoning here, especially since this is a write operation that only we will run (and it only needs to be ran a few times)